src/screens/Noticeboard/NoticeboardSuccessScreen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { BoldText, Wrapper } from '../../components';
+
+export const NoticeboardSuccessScreen = ({ route }: { route: any }) => {
```

Comment:
we do not need this screen anymore as we need to go back to the previous screen in case of a new entry or stay on the same in case of a message (with reseted form)

Reply:
19fdbfe

Reply:
As previously discussed, it is written according to the PDF on the ticket. This screen was not available in the previous version.

It can be removed.